ios/BetterSocial.xcodeproj/project.pbxproj

```diff
@@ -351,7 +350,7 @@
 			isa = PBXFrameworksBuildPhase;
 			buildActionMask = 2147483647;
 			files = (
-				C36230AE8C5C3DD90FCED3EC /* BuildFile in Frameworks */,
+				C36230AE8C5C3DD90FCED3EC /* (null) in Frameworks */,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

These lines also have changes from `BuildFile` to `(null)` in the Frameworks section. As mentioned before, please ensure that these changes are intentional and won't lead to missing dependencies during build time.

```diff
- /* BuildFile in Frameworks */
+ /* (null) in Frameworks */
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->